apps/api/src/utils/config-manager.ts

```
@@ -0,0 +1,539 @@
+      default:
+        return true
+    }
+  }
+
+  private parseEnvironmentValue(value: string, type: ConfigSchema['type']): any {
+    switch (type) {
+      case 'string':
+        return value
+      case 'number':
+        const num = Number(value)
+        if (isNaN(num)) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)